generators/app/templates/src/config/navigation.ejs

```diff
@@ -0,0 +1,77 @@
+<%_ if (features.drawerandroid || features.drawerios) { _%>
+import React from 'react';
+import { Button } from 'react-native';
+
+<%_ } _%>
+import * as Routes from '../constants/routes';
+import { blue, white } from '../constants/colors';
+<%_ if (features.drawerandroid || features.drawerios) { _%>
+import { actionCreators as drawerActions } from '../redux/drawer/actions';
+<%_ } _%>
+
+import I18n from './i18n';
+
+// Default nav options for all screens
+const defaultNavOptions = ({ navigation }) => ({
+  // Change screen title from i18n traslates files
+  title: I18n.t(`app:${navigation.state.routeName}`),
+
+  // TODO: The following options are examples. Change them to your need
+  headerStyle: {
+    backgroundColor: blue
+  },
+  headerBackTitleStyle: {
+    color: white
+  },
+  headerTitleStyle: {
+    color: white
+    // TODO: If you want to change the header font, apply these options
```

Comment:
I'd add a reference to https://medium.com/react-native-training/react-native-custom-fonts-ccc9aacf9e5e